src/js/components/FileInput/doc.js

```diff
@@ -0,0 +1,47 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge, themeDocUtils } from '../../utils';
+
+export const doc = FileInput => {
+  const DocumentedFileInput = describe(FileInput)
+    .availableAt(getAvailableAtBadge('FileInput'))
+    .description('A control to input one or more files.')
+    .usage(
+      `import { FileInput } from 'grommet';
+<FileInput id='file' name='file' />`,
+    )
+    .intrinsicElement('input');
+
+  DocumentedFileInput.propTypes = {
+    id: PropTypes.string.description('The id attribute of the input.'),
```

Comment:
why do we need to document `id` for this component, while it is not documented for others?

Reply:
Just following what we do for `TextInput`.